### PR TITLE
feat-support multiple params with comma after -p 

### LINKED ISSUE #1388: [Feature][core] support muti params after -p
### Search before asking

- [X] I had searched in the [issues](https://github.com/DTStack/chunjun/issues) and found no similar feature requirement.

### Description

As we know we can set params after -p like this 
```
-p mysql_server=127.0.0.1
```
Then we can use it in our json file like this
```
${mysql_server}
```

But we don't support -p with comma like this
```
-p bootstrap=host1:9092,host2:9092
```

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liumengkai**
## Purpose of this pull request
support multiple params after -p ，If we want set multiple params with comma,we can set like this:
```
-p -Mbootstrap=host1:9092,host2:9092
```

## Which issue you fix
#1388

## Checklist:

- [x] I have executed the **'mvn spotless:apply'** command to format my code.
- [ ] I have a meaningful commit message (including the issue id, **the template of commit message is '[label-type-#issue-id][fixed-module] a meaningful commit message.'**)
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have checked my code and corrected any misspellings.
- [ ] My commit is only one. (If there are multiple commits, you can use 'git squash' to compress multiple commits into one.)

### CODE REVIEW

**Comment by liumengkai** [comment]
code review
@FlechazoW

File: chunjun-core/src/main/java/com/dtstack/chunjun/util/JobUtil.java [hunk 1]
```diff
@@ -48,13 +48,29 @@ public static String JsonValueReplace(String json, Map<String, String> parameter
...
+     * @description: convert params after '-p' in shell to hashmap
+     */
     public static HashMap<String, String> CommandTransform(String command) {
         HashMap<String, String> parameter = new HashMap<>();
         String[] split = StringUtils.split(command, ConstantValue.COMMA_SYMBOL);
-        for (String item : split) {
-            String[] temp = item.split(ConstantValue.EQUAL_SYMBOL);
-            parameter.put(temp[0].trim(), temp[1].trim());
+        for (int i = 0; i < split.length; i++) {
+            String[] temp = split[i].split(ConstantValue.EQUAL_SYMBOL);
```
Comment (FlechazoW) [inline chunjun-core/src/main/java/com/dtstack/chunjun/util/JobUtil.java:60]: I don't believe that a parameter named 'temp' is readable.

Comment (liumengkai) [reply chunjun-core/src/main/java/com/dtstack/chunjun/util/JobUtil.java:60]: agree. I will make a change

**Review by FlechazoW — APPROVED** [review]
LGTM

**Comment by liumengkai** [comment]
json paramters will be great.
more information: #1393

**Pull request closed without merging**